source

```diff
@@ -103558,7 +103558,8 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
     <p>If no match can be made and the <var data-x="temporary buffer">temporary buffer</var>
     consists of a U+0026 AMPERSAND character (&amp;) followed by a sequence of one or more <span
     data-x="ASCII alphanumeric">ASCII alphanumerics</span> and a U+003B SEMICOLON character (;),
-    then this is a <span>parse error</span>.</p>
+    then this is an <dfn data-x="parse-error-invalid-character-reference-name">invalid-character-reference-name</dfn>
```

Comment:
This error happens when you have named character reference (like `&amp;`) but with unknown code (e.g. `&somethingUnknown;`), so I guess it's better be named `unknown-named-character-reference`